Automattic/wp-calypso
# Add coffee color scheme from wp-admin

One part of #47036

#### Changes proposed in this Pull Request

* Add coffee color scheme

#### Testing instructions

* Visit /me/account
* Choose coffee and save

To see the wp-admin color scheme, create an atomic site, then visit /wp-admin/profile.php, choose color scheme and save.

Calypso:
![2020-11-05_17-14](https://user-images.githubusercontent.com/937354/98307130-b5bde100-1f8a-11eb-88b9-060070ee9c9b.png)

Wp-admin:
![2020-11-05_17-13](https://user-images.githubusercontent.com/937354/98307135-b9e9fe80-1f8a-11eb-96cb-4eec6617e3ce.png)

#### Notes

If you're following along with Ectoplasm from #47122, you'll see I generated a palette of "Ectoplasm green" in -10 to -90 values.  For this one, I found that "Studio Orange" was a close enough match for primary+accent colors and reused it.  Orange and brown are close to each other in terms of color hue, who would have thunk it?  I'm open to revision or ideas on this, though.

In wp-admin this is a "low contrast" scheme, which means that certain buttons are black instead of the highlight color.  I haven't accounted for that here.
